pfcpiface/parse-far_test.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022-present Open Networking Foundation
+
+package main
+
+import (
+	"net"
+	"testing"
+
+	"github.com/omec-project/upf-epc/test/integration"
+)
+
+func TestFARParsing(t *testing.T) {
```

Comment:
Since it's a first unit test and others might follow the convention in future, I suggest to apply Table-based testing at the very beginning. It's a common test pattern in Go: https://medium.com/agrea-technogies/basic-testing-patterns-in-go-d8501e360197

Reply:
```suggestion
func TestParseFAR(t *testing.T) {
```

Reply:
Done. I added a couple of table driven tests, one testing valid input and the other invalid input.

In principle we could run all the tests with only one big table but I think it's better to have multiple table driven tests.
Please let me know if you think otherwise

Reply:
I like the current shape.